src/argon2.c

```diff
@@ -175,15 +175,15 @@ int argon2_hash(const uint32_t t_cost, const uint32_t m_cost,
 
     /* Detect and reject overflowing sizes */
     /* TODO: This should probably be fixed in the function signature */
```

Comment:
I left this TODO because it was pre-existing, but I'm not sure there's a valid TODO remaining.